sdk/python/feast/infra/offline_stores/bigquery.py

```diff
@@ -571,12 +573,22 @@ def to_remote_storage(self) -> List[str]:
 
         table = self.to_bigquery()
 
+        parquet_file_size_mb = os.getenv("BQ_EXPORT_PARQUET_FILE_SIZE_MB")
+        if parquet_file_size_mb is not None:
+            table_size_in_mb = self.client.get_table(table).num_bytes / 1024 / 1024
```

Comment:
Do I understand correctly that this only works if we'll be sending all of the data in the table? If so, I'm not sure this is a good approach, since we'll almost always be sending only a small portion of data.

Reply:
Maybe I am wrong, but I create a temporary table with the latest entity rows and export it. So I don't understand what you meant by `sending all of data` and `almost always be sending only a small portion of data`?

Reply:
If `table` here is really the temporary tables that only contains the latest rows, then this is fine 😁 From first glance it seemed like this might be the "original" table

Reply:
[Here](https://github.com/vinted/feast/blob/091b7b1e66c8a1c65874512591183da1351d7f19/sdk/python/feast/infra/materialization/contrib/bytewax/bytewax_materialization_engine.py#L185-L196) is the code where it executes to remote storage. So if I understand correctly then it is not exporting full table 😊